### ISSUE TITLE
Release a stable v1

### ISSUE THREAD

**Issue by roquie** [HarmonyIO/Validation#73]
Hi! This library is so cool for async validation in PHP. Can I help you to release a stable version?

**Comment by PeeHaa** [comment]
As far as I am concerned the public API works pretty well and I don't see an issue releases a stable v1

**Comment by roquie** [comment]
Good. May be used stable versions for these dependencies?

https://github.com/HarmonyIO/Validation/blob/master/composer.json#L12
https://github.com/HarmonyIO/Validation/blob/master/composer.json#L33
https://github.com/HarmonyIO/Validation/blob/master/composer.json#L34

Because simple command `composer require harmonyio/validation` does not work without hard-installed dependencies with RC-versions, like this:

```
# composer `require` section 
"harmonyio/validation": "v1.0.0-rc1",
    "harmonyio/cache": "v1.0.0-rc1",
    "harmonyio/http-client": "v1.0.0-rc1"
```

Needs to be release these dependencies with stable versions and then update this library to 1.0.0. Please :)

**Comment by PeeHaa** [comment]
Good call. Will review all dependencies and update them where needed before releasing

**Comment by roquie** [comment]
Do you have a time to update composer dependencies for this library? And also update "dependencies of dependencies" (`harmony/cache`, `harmony/http-client`)?

Because amphp team released new http client and in my project it conflict with old artax version in your http-client: https://github.com/HarmonyIO/Http-Client/blob/master/composer.json#L18 ?

I was glad to make a pull-request, but composer dependencies should be resolved by author. :(

**Comment by roquie** [comment]
IMHO, `harmony/cache`, `harmony/http-client` must be in composer section `suggests`, because caching/rules-with-http-requests is optional functionality.

**Comment by roquie** [comment]
Fuh. I'm completed work.

https://github.com/spacetab-io/harmony-validation-php
https://github.com/spacetab-io/harmony-cache-php
https://github.com/spacetab-io/harmony-http-client-php

`composer require spacetab-io/harmony-validation`

Works pretty well.
